### ISSUE TITLE
Documentation Discrepancies

### ISSUE THREAD

**Issue by MingyJ** [Kovah/LinkAce#83]
I mean this with all due respect, I think the project looks awesome! It's super hard to follow the directions though! 

https://github.com/Kovah/LinkAce/#setup-with-docker

https://www.linkace.org/docs/v1/setup/setup

Those two links give different directives than one another and also cause you to expect different things to happen i.e. docker container with php is in one set of docs expected to be "linkace_php_1" and in the next "linkace-php" both of which I was able to produce depending. 

The biggest hiccup is in /setup/setup there is no mention of renameing .env.docker to .env but in #setup-with-docker it does. 

I know the app is in Dev. so it's cool, I just wanted you to know what's happening is all :)

**Comment by Kovah** [comment]
Thanks for the info, will take a look into this shortly

**Comment by Kovah** [comment]
Hi Brett,
so, the currently most actual version is the wiki on linkace.org. I updated the [readme](https://github.com/Kovah/LinkAce/blob/dev/README.md) in the dev version accordingly and moved some party completely into the wiki.
Could you share some more information about the problems you had? It's not that easy to write the docs and every hint for possible improvements could help me a lot.
Thank you very much!

**Comment by Drallas** [comment]
Reading the docs careful, I found some reference to linkace_php_1 at [CLI Commands for LinkAce](https://www.linkace.org/docs/v1/cli/) let me know if you want them changed to linkace_app_1? 

btw where is this linkace_php_1 / linkace_app_1 string derived from /app /php ?

**Comment by Kovah** [comment]
@Drallas Of couse, please do so.
linkace_app_1 is derived from the naming of the Docker container based on the docker-compose.yml files.
